tests/payment_methods_test.go

```diff
@@ -8,5 +8,4 @@ func (c *ClientTests) TestPaymentMethodsAll() {
 	require.NoError(err)
 
 	assert.True(paymentMethods.PrimaryPaymentMethod != nil)
-	assert.True(paymentMethods.SecondaryPaymentMethod != nil)
```

Comment:
I am not sure why this assertion is removed from this PR?

Reply:
The cassette does not have a secondary payment method in it.

Reply:
The purpose of these assertions is to ensure the keys exist. We may need to adjust it to account for keys only and not values like we did with one of the other libs.

Reply:
Re-added line, tweaked a bit to account for the cassette